managed-ledger/src/main/java/org/apache/bookkeeper/mledger/intercept/ManagedLedgerInterceptor.java

```diff
@@ -36,9 +36,8 @@
      * Intercept an OpAddEntry and return an OpAddEntry.
      * @param op an OpAddEntry to be intercepted.
      * @param numberOfMessages
-     * @return an OpAddEntry.
      */
-    OpAddEntry beforeAddEntry(OpAddEntry op, int numberOfMessages);
+    void beforeAddEntry(OpAddEntry op, int numberOfMessages);
```

Comment:
please do not change public APIs

Reply:
It's new API introduced in master, we have not released it yet.